### ISSUE TITLE
Preserve RNG starting seed with !restartemu

### ISSUE THREAD

**Issue by MathyFurret** [TwitchPlaysPokemon/public#124]
This is preferable when the game crashes to make sure the same game is played. Both the game's RNG and the move selector's RNG need to be restored when the match is restarted.

EDIT: Some testing will need to be done to see if identical results can actually be obtained with this method, otherwise a different solution should be found

**Comment by Felk** [comment]
I disagree that this should be done. If a game plays for a couple of rounds and it gets restarted, being able to predict what would happen in the first few rounds sounds like a bad thing. for me the goal is equal opportunity, which you have with random rng each time, and not with fixed rng

**Comment by MathyFurret** [comment]
resolved in discord

> yes, only if players can influence it
> if additional knowledge cannot influence it, fixing rng would be fine
